cocos/core/utils/prefab/prefab-info.ts

```diff
@@ -139,6 +139,9 @@ export class PrefabInstance {
     @type([TargetInfo])
     public removedComponents: TargetInfo[] = [];
 
+    @serializable
```

Comment:
Please add a comment to explain the usage, in addition, please change the above Chinese comments to English